apps/asap-server/test/controllers/teams.test.ts

```diff
@@ -46,18 +42,12 @@ describe('Team controller', () => {
     algoliaApiKey: 'test-api-key',
   };
 
-  const getUserFilterExpectation = (teamId: string): string =>
-    `data/teams/iv/id eq '${teamId}' and data/onboarded/iv eq true`;
-
   beforeAll(() => {
     identity();
   });
 
   afterEach(() => {
     expect(nock.isDone()).toBe(true);
-  });
```

Comment:
why are you merging the two `afterEach`? they are here for a reason and the reason is that if it's a single `afterEach` once the first statement throws the other one will not get executed - but we need to clean all mocks each time so it's important that it runs. It can't be run as first however as it will mess up the expectation

Reply:
No reason. I misunderstood this. Thanks for the explanation